### PR TITLE
Remove unsupported playbooks and utilities

### PR DESCRIPTION
**Pull request by mtnbikenc**
- Removes `playbooks/{aws,gce,libvirt,openstack}` (with the exception of new provisioning aws playbooks)
- Removes `inventory/{aws,gce,libvirt,openstack}`
- Removes `bin/cluster`
- Updates docs with removal of above

### CODE REVIEW

**Comment by sdodson** [comment]
Need @kwoodson to make sure we're not nuking his stuff. Otherwise fine with me.

**Comment by mtnbikenc** [comment]
@sdodson Yeah, I saw @kwoodson 's changes merge but then didn't leave his stuff.  I'll update.

**Comment by mtnbikenc** [comment]
@kwoodson I've updated the PR and hopefully have not deleted any of your recent provisioning work.  PTAL

File: README.md [hunk 1]
```diff
@@ -69,9 +69,6 @@ you are not running a stable release.
 
 2. Setup for a specific cloud:
```
Comment (michaelgugino) [inline README.md]: nit:  This heading no longer makes much sense.

File: inventory/README.md [hunk 2]
```diff
@@ -2,8 +2,4 @@
 
 You can install OpenShift on:
 
-* [Amazon Web Services](aws/hosts/)
 * [BYO](byo/) (Bring your own), use this inventory config file to install OpenShift on your bare metal servers
```
Comment (michaelgugino) [inline inventory/README.md]: nit: 'bare metal or virtualized/cloud servers'

Comment (sdodson) [inline inventory/README.md]: pre-existing hosts?

**Comment by mtnbikenc** [comment]
aos-ci-test

**Comment by mtnbikenc** [comment]
Not sure what happened there.  `OS_3.6_containerized` passed then five minutes later is says failed, but the test passed in the logs.  :confused:

**Comment by mtnbikenc** [comment]
aos-ci-test

**Comment by mtnbikenc** [comment]
@cooktheryan I know @sdodson 's PR to contrib already merged.  Is there anything else here that I'm removing that should be moved over to contrib which was not part of his original PR? (`README_*.md`, `inventory/{aws,gce,libvirt,openstack}`, `common/*/service.yml`)

**Comment by sdodson** [comment]
[merge]

**Comment by sdodson** [comment]
If there are missing changes we'll just pull them out of history prior to this merge.

**Comment by cooktheryan** [comment]
I see no issues from my end

**Comment by mtnbikenc** [comment]
Test succeeded.  Merge failed on an unrelated tox flake8 failure.  All integration tests passed.  Merging.

**Pull request merged**